app/controllers/dogs_controller.rb

```diff
@@ -95,11 +95,9 @@ def update
   end
 
   def create
-    @foster_users = User.all
     @dog = Dog.new(dog_params)
-    if @dog.tracking_id.blank?
-      @dog.tracking_id = Dog.connection.select_value("SELECT nextval('tracking_id_seq')")
-    end
+    @dog.tracking_id = Dog.next_tracking_id if @dog.tracking_id.blank?
```

Comment:
this could be moved to a default value in the DB instead of the application having to worry about it, maybe make a ticket?